### PR TITLE
Introduce new Used By section

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- https://github.com/kyma-project/website/issues/630

**Related issue(s)**

https://github.com/kyma-project/website/issues/630

### CODE REVIEW

**Comment by valentinvieriu** [comment]
Looks almost done. I would recommend removing the auto play on mobile. 
As we try to simplify things on our website too, one idea I've had was to use something like [scroll-snap](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-snap-type) for the supporting browsers and normal scrolling for the rest. This is just an idea

**Comment by aerfio** [comment]
Note to self: remeber about docs, checking SEO/lighthouse and that bug with resizing

File: src/views/landingPage/UsedBy/index.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+const AddCompanyButton = () => (
+  <Link.External to={config.links.ADD_KYMA_USER}>
+    <Button.Emphasized size="md">
+      <FormattedMessage id={gt("addYourCompany")} />
+    </Button.Emphasized>
+  </Link.External>
+);
+
+const ButtonWrapper = styled.div`
```
Comment (tgorgol) [inline src/views/landingPage/UsedBy/index.tsx:104]: Shouldn't these styled components be moved to `styled.tsx` file?

File: src/views/landingPage/UsedBy/index.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+              <HeaderWrapper marginBottom={10}>
+                <H as="h2">
+                  <FormattedMessage
+                    id={gt("headline")}
+                    tagName={React.Fragment}
+                  />
+                </H>
+                <FormattedMessage tagName="p" id={gt("paragraph")} />
+              </HeaderWrapper>
+              <MobileGallery customers={adopters} isMobile={isMobile} />
```
Comment (tgorgol) [inline src/views/landingPage/UsedBy/index.tsx:60]: Were the cards supposed to have fixed height in mobile view?

File: src/views/landingPage/UsedBy/index.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  </Link.External>
+);
+
+const ButtonWrapper = styled.div`
+  margin-top: 50px;
+  display: flex;
+  justify-content: center;
+  /* & button {
+    padding: 0 30px;
+  } */
```
Comment (tgorgol) [inline src/views/landingPage/UsedBy/index.tsx:110]:
```suggestion
```

**Review by tgorgol — COMMENTED** [review]
Good job, looks really nice! Remember that you can use [this PR](https://github.com/kyma-project/website/pull/627) for previews so that we don't accidentally merge some breaking changes to the current website :)

File: src/views/landingPage/UsedBy/index.tsx (hunk 2 above)
Comment (aerfio) [reply src/views/landingPage/UsedBy/index.tsx:60]: Technically yes, but it's not possible to do in the library we are using (and I personally think that those cards cannot have the same height, imagine a situation in which there's one card that is very tall, and it's not visible, but it's height is altering the height of currently visible ones)

File: src/views/landingPage/UsedBy/index.tsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                  <FormattedMessage
+                    id={gt("headline")}
+                    tagName={React.Fragment}
+                  />
+                </H>
+                <FormattedMessage tagName="p" id={gt("paragraph")} />
+              </HeaderWrapper>
+            </Grid.Unit>
+            <CustomerPair customers={customers.slice(0, 2)} />
+            {!!opened ? <CustomerPair customers={customers.slice(2)} /> : null}
```
[the remark this answers is not in the document]
Comment (aerfio) [reply src/views/landingPage/UsedBy/index.tsx:77]: Yup, but it has been discussed with @valentinvieriu, it's ok for him

**Review by valentinvieriu — APPROVED** [review]

**Review by tgorgol — APPROVED** [review]

**Review by m00g3n — DISMISSED** [review]

File: content/i18n/en/landing-page.json [hunk 5]
```diff
@@ -31,18 +31,37 @@
...
-        "paragraphs": ["Kyma ensures smooth Day-2 support for the standalone applications and extensions you'll create. Thanks to numerous CNCF open-source projects that come preconfigured with Kyma, you can easily monitor the performance of your microservices and functions, scan logs for errors, and upgrade your solutions painlessly."]
+        "paragraphs": [
+          "Kyma ensures smooth Day-2 support for the standalone applications and extensions you'll create. Thanks to numerous CNCF open-source projects that come preconfigured with Kyma, you can easily monitor the performance of your microservices and functions, scan logs for errors, and upgrade your solutions painlessly."
+        ]
       }
     }
   },
+  "usedBy": {
+    "headline": "Used by",
+    "paragraph": "Discover the main usecases of our happy customers",
```
Comment (aerfio) [inline content/i18n/en/landing-page.json:55]:
```suggestion
    "paragraph": "Discover the main use cases of our happy customers",
```

**Review by kazydek — APPROVED** [review]
Approving as texts for the adopters will be submitted later.

**Review by m00g3n — APPROVED** [review]

**Pull request merged**